src/commander.ts

```diff
@@ -72,6 +72,24 @@ export class Commander {
         this.extension.completer.citation.browser()
     }
 
+    binaries() {
+        function windowsPaths(command: string, disks: string[]=['C', 'D'], tlVers: string[]=['2014', '2015', '2016', '2017']) {
```

Comment:
I would be tempted to maybe use globs to search for paths to avoid having these things hardcoded. If so, something like [`glob`](https://www.npmjs.com/package/glob) (`npm install --save glob && npm install --save-dev @types/glob`) could be used:

```typescript
import * as glob from 'glob'
 
glob("${disk}:\\TexLive\\*\bin\${command}.exe", (er, files) => {
  // files is an array of filenames. 
})
```

This will find anything matching the pattern.